src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -14,9 +17,12 @@ module.exports = Model.extend({
   },
 
   url: function () {
-    var params = [
-      'bins=' + this.get('bins')
-    ];
+    var params = [];
+    if (this.get('column_type') === 'date' && this.get('aggregation')) {
+      params.push('aggregation=' + this.get('aggregation'));
+    } else if (this.get('bins')) {
```

Comment:
some times we do the double check if has the attribute, and also if column type is number, doesn't this apply here?